assets/common/scripts/game_manager.gd

```diff
@@ -42,47 +30,52 @@ func _shortcut_input(event):
 		increment_day()
 		get_viewport().set_input_as_handled()
 
+#region Zones
+var current_zone: Zone
+
 func register_zone(zone: Zone):
 	current_zone = zone
-	if not Savegame.player.zones.has(zone.id):
-		Savegame.player.zones[zone.id] = init_map()
+	if not Savegame.player.grid.has(zone.id):
+		Savegame.player.grid[zone.id] = init_map()
 	current_zone_updated.emit()
 
 func deregister_zone(zone: Zone):
 	if zone == current_zone:
 		current_zone = null
 		current_zone_updated.emit()
+#endregion
+
+#region Grid
+func get_grid():
```

Comment:
Same as all the other comments, this makes me think it'll return me the grid node